src/components/SurveyAnswer/ChoiceAnswer.tsx

```diff
@@ -7,10 +7,10 @@ import 'simplebar-react/dist/simplebar.min.css';
 const ChoiceAnswer = ({
```

Comment:
Note to reviewer: All of the changes in the SurveyAnswer component are to rename prop